test/e2e/specs/help-center/help-center__general-interaction.ts

```diff
@@ -2,11 +2,141 @@
  * @group calypso-pr
  */
 
-import { SupportComponent, TestAccount } from '@automattic/calypso-e2e';
+import {
+	DataHelper,
+	SupportComponent,
+	TestAccount,
+	TestAccountName,
+} from '@automattic/calypso-e2e';
 import { Browser, Page } from 'playwright';
 
 declare const browser: Browser;
 
+/** Tests to ensure the Help Center is open and visible in Calypso and the Editor */
+describe.each( [
+	{ accountName: 'defaultUser' as TestAccountName },
+	{ accountName: 'atomicUser' as TestAccountName },
+] )( 'Help Center: Verify Help Center is accessible', function ( { accountName } ) {
+	let page: Page;
+	let supportComponent: SupportComponent;
+	let testAccount: TestAccount;
+	const helpCenterContainerVisibilityErrorMessage = `This is a bug that should be urgently fixed.
+	But because this test runs against ETK production, this bug was probably not introduced in this pull request.
+	Please consider alerting the last person who deployed ETK to attend to this issue and fix the Help Center.`;
```

Comment:
```suggestion
	const helpCenterContainerVisibilityErrorMessage = `This is a bug that should be urgently fixed.
	But because this test runs against ETK production, this bug was probably not introduced in this pull request.
	Please consider alerting the last person who deployed ETK to attend to this issue and fix the Help Center. You can view the history here: https://github.com/Automattic/wp-calypso/commits/trunk/apps/editing-toolkit`;
```